apps/core/scenes/search/Search.js

```diff
@@ -22,12 +22,15 @@ export type SearchParameters = {|
   +tripType: string,
   +travelFrom: ?$ReadOnlyArray<Location>,
   +travelTo: ?$ReadOnlyArray<Location>,
-  +sort: string,
+  +sortBy: string,
```

Comment:
Pls change this name also in the `parseURLqueryToState` (PARSER_CONFIG).
`/margarita/apps/core/contexts/searchContext/helpers.js`

Reply:
Changed